src/coprocessor/codec/mysql/duration.rs

```diff
@@ -10,8 +10,7 @@ use tikv_util::codec::BytesSlice;
 
 use nom::character::complete::{digit1, multispace0, multispace1};
 use nom::{
-    alt_complete, call, char, complete, cond_with_error, do_parse, eof, map, map_res, opt, peek,
-    preceded, tag, IResult,
+    alt, call, char, complete, cond, do_parse, eof, map, map_res, opt, peek, preceded, tag, IResult,
```

Comment:
`use nom::*`? Or prefer to just use `nom::xxx` everywhere to be clear (since `char`, `map` are likely to cause confuse).

Reply:
`map` has been imported as a macro by another crate globally, so I will fix it in the latter manner.